src/actions/domUtils.js

```diff
@@ -0,0 +1,17 @@
+export const getStackDomain = () => {
+  if (document.querySelector('[role=application]')) {
+    return (
+      '//' + document.querySelector('[role=application]').dataset.cozyDomain
+    )
+  } else {
+    return null
+  }
+}
+
+export const getStackToken = () => {
+  if (document.querySelector('[role=application]')) {
+    return '//' + document.querySelector('[role=application]').dataset.cozyToken
```

Comment:
I don't think we should have `'//' +` for the token

Reply:
Fixed